docs/features/battery.mdx

```diff
@@ -79,6 +79,23 @@ Wie auch im vorherigen Beispiel endet die Ladung, wenn der Ladestand des Hausspe
   caption="Automatischer Start"
 />
 
+## Batterie Boost
+:::warning Experimentell
+Batterie Boost befindet sich gerade noch in der Erprobungsphase.
+:::
+
+Durch den Batterie Boost wird der PV-Modus maximal durch die Hausbatterie unterstützt. Zusätzlich zu der Leistung aus der Photovoltaikananlage wird die maximal mögliche Leistung aus der Batterie gezogen. Das Boostverhalten endet, wenn das Auto abgesteckt wird oder der Modus umgeschaltet wird. 
+
+Der Batterie Boost kann in den Einstellungen des Loadpoints aktiviert werden:
+
+<Screenshot
+  name="features/screenshots/battery-boost"
+  caption="Batterie Boost aktivieren"
+/>
+
+Beispiel:
+Dies ist nützlich wenn das Fahrzeug in Kürze die Ladestation verlassen wird, in den kommenden Stunden aber viel Sonnenüberschuss verfügbar sein wird. So kann kurz vor der Abfahrt noch Energie aus der Hausbatterie ins Fahrzeug geladen werden und die Hausbatterie lädt sich danach wieder auf.
```

Comment:
```suggestion
Dies ist nützlich wenn das Fahrzeug in Kürze die Ladestation verlassen wird, in den kommenden Stunden aber viel Sonnenüberschuss verfügbar sein wird.
So kann kurz vor der Abfahrt noch Energie aus der Hausbatterie ins Fahrzeug geladen werden und die Hausbatterie lädt sich danach wieder auf.
```

Bzgl. Formatierung haben wir die Regel, dass ein Satz immer in einer eigenen Zeile steht. Für die Ausgabe ist das egal, aber das macht die Diffs bei Änderung deutlich besser lesbar.
Betrifft auch die anderen Absätze hier im PR.

Reply:
Alles klar, danke dass versteh ich